lib/std/math/isnormal.zig

```diff
@@ -1,45 +1,43 @@
 const std = @import("../std.zig");
 const math = std.math;
 const expect = std.testing.expect;
-const maxInt = std.math.maxInt;
 
 // Returns whether x has a normalized representation (i.e. integer part of mantissa is 1).
```

Comment:
This should have a better explanation or at least mention the standard paper ["What Every Computer Scientist Should Know About Floating-Point Arithmetic" by David Goldberg](https://docs.oracle.com/cd/E19957-01/806-3568/ncg_goldberg.html#936).

If you feel fancy add the [most relevant definition bits](https://www.volkerschatz.com/science/float.html) to math.zig that are missing:`MANT_DIG, MIN_EXP and MAX_EXP`.

```txt
EPSILON | 0.0625 | Difference between 1 and next larger representable value
-- | -- | --
MIN | 0.25 | Smallest normalised representable value
MAX | 15.5 | Largest positive representable value
MANT_DIG | 5 | Mantissa bit size including implied leading 1
MIN_EXP | -2 | Minimum exponent for normalised number; also implied for subnormals
MAX_EXP | 3 | Maximum exponent
TRUE_MIN | 0.015625 | Smallest positive representable value
```